torchvision/models/detection/anchor_utils.py

```diff
@@ -164,8 +162,8 @@ def __init__(
         clip: bool = True,
     ):
         super().__init__()
-        if steps is not None and len(aspect_ratios) != len(steps):
-            raise ValueError("aspect_ratios and steps should have the same length")
+        if steps is not None:
+            assert len(aspect_ratios) == len(steps)
```

Comment:
Why we need to undo this? It's part of the constructor where usually we don't care to be JIT-scriptable or FX-traceable. Do we initialize this directly in a forward?

Reply:
you are right, I think I mess this one up